helm/manager/Chart.yaml

```diff
@@ -1,15 +1,16 @@
-apiVersion: v1
-description: A Helm chart for Kubernetes
+apiVersion: v2
+description: Hydro-serving-manager helm chart
```

Comment:
A helm chart for the service which manages serving related entities for the entire system